### PR TITLE
404 [en, ru], edit [en, ru], keeping upd, fixes

### PR DESCRIPTION
**Pull request by 0x22almostEvil**
404 [404.tsx, en, ru -> common.json],
edit [TaskControls.tsx, en, ru -> common.json]
Fixed leaderboard.json for missed PR,
Better context for ru,
Fixed typos for ru

### CODE REVIEW

**Comment by 0x22almostEvil** [comment]
Finally (Sorry, Prettier wasn't working well for me for some reason)

File: website/src/components/Survey/TaskControls.tsx [hunk 1]
```diff
@@ -53,7 +53,7 @@ export const TaskControls = ({
             </>
           ) : (
             <>
-              <Tooltip label="Edit">
+              <Tooltip label={t("edit")}>
                 <IconButton size="lg" data-cy="edit" aria-label="edit" onClick={onEdit} icon={<Edit2 size="1em" />} />
```
Comment (notmd) [inline website/src/components/Survey/TaskControls.tsx:57]: Can you update the aria-label too?

Comment (0x22almostEvil) [reply website/src/components/Survey/TaskControls.tsx]: Sure

**Review by notmd — APPROVED** [review]
thanks

**Pull request merged**